conan/cli/commands/graph.py

```diff
@@ -83,6 +85,37 @@ def graph_build_order(conan_api, parser, subparser, *args):
     return install_order_serialized
 
 
+@conan_subcommand(formatters={"text": cli_out_write})
+def graph_build_order_prepare(conan_api, parser, subparser, *args):
+    """
+    Compute the build order of a dependency graph.
+    """
+    subparser.add_argument("build_order_file", help="build-order json file")
+    subparser.add_argument("ref", help="recipe to build")
+    subparser.add_argument("package_id", help="package to build")
```

Comment:
Couldn't we put this two arguments together so that you need to pass `pkg/1.0:abc123`? Your way is more explicit which might avoid confusion though..

Reply:
Yes, that is doable, whatever UI the team prefers.

Reply:
Yes, I think I agree with adding the package_id in the reference too instead of a separate argument